bot/bot.py

```diff
@@ -462,50 +459,48 @@ def join_event_handler(
     def text_handler(
         self,
         update: Update,
-        _: CallbackContext
+        _: CallbackContext,
     ) -> NoReturn:
+        # let's use integer division and forget about cents for some time
         expense_sum = update.effective_message.text
         user_id = update.effective_user.id
         try:
-            expense_sum = float(expense_sum)
+            expense_sum = int(expense_sum)
         except ValueError:
-            update.effective_chat.send_message('Потраченная сумма вводится в формате: 123(целое число) или '
-                                               '456.23(дробная число, целая часть от дробной отделяется точкой. '
+            update.effective_chat.send_message('Потраченная сумма вводится в формате: 123(целое число). '
                                                'Попробуй еще раз')
             return
 
-        self.bot_.expenses[user_id]["sum"] = expense_sum
-        expense = self.bot_.expenses[user_id]
-        expense_id = self.bot_.splitwise.add_expense(expense['name'], user_id, expense['token'], expense['sum'])
-        self.bot_.expenses[user_id]["id"] = expense_id
+        expense_id = self.bot_.splitwise.add_expense(self.bot_.expenses[user_id])
```

Comment:
Hello, Alex! How are you doing? Hope you don't mind specifying expense_sum before persisting it to the db

Reply:
Hello, Dmitry! Thank you very much for this issue! I will fix it as soon as possible.